plugins/auth/index.js

```diff
@@ -178,12 +178,18 @@ exports.register = (server, options, next) => {
                                         const description =
                                             `The default collection for ${user.username}`;
 
-                                        collectionFactory.get({
-                                            userId: user.id,
-                                            type: 'default'
+                                        collectionFactory.list({
+                                            params: {
+                                                userId: user.id,
+                                                type: 'default'
+                                            }
                                         })
-                                            .then((collection) => {
-                                                if (!collection) {
+                                            .then((collections) => {
+                                                const defaultCollection = collections
```

Comment:
probably can just take the first collection returned
```suggestion
                                                const defaultCollection = collections[0];
```